src/utils/bitmap_from_file.hpp

```diff
@@ -31,10 +31,13 @@
 
 #include "utils/bitmap.hpp"
 
+struct BGRColor;
+
 Bitmap load_error_bitmap();
 
 /// \return an invalid bitmap if error
-Bitmap bitmap_from_file_impl(const char * filename);
+Bitmap bitmap_from_file_impl(const char * filename,
+                             const BGRColor * = nullptr);
 
 /// \return always a valid bitmap
-Bitmap bitmap_from_file(const char * filename);
+Bitmap bitmap_from_file(const char * filename, const BGRColor * = nullptr);
```

Comment:
Name of the new parameter should be more explicit, like `background_color` or `bg_color`.
Same for the other function `bitmap_from_file*`